src/Utilities/Compiler/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -208,35 +193,21 @@ static bool TryParseValue(string value, out ImmutableHashSet<TEnum> result)
             DiagnosticDescriptor rule,
             SyntaxTree tree,
             Compilation compilation,
-            bool defaultValue,
-            CancellationToken cancellationToken)
+            bool defaultValue)
         {
-            var analyzerConfigOptions = options.GetOrComputeCategorizedAnalyzerConfigOptions(compilation, cancellationToken);
+            var analyzerConfigOptions = options.GetOrComputeCategorizedAnalyzerConfigOptions(compilation);
             return analyzerConfigOptions.GetOptionValue(optionName, tree, rule, bool.TryParse, defaultValue);
         }
 
-        public static uint GetUnsignedIntegralOptionValue(
```

Comment:
FYI: I kept these unused overloads for consistency with other APIs in this file (one symbol overload and one syntax tree overload). It is fine to delete them though.